soccer/gameplay/tactics/wall.py

```diff
@@ -78,17 +79,19 @@ def on_enter_defense_wall(self):
                 priority=priority)
 
     def on_enter_scramble(self):
-        self.number_of_defenders = self.number_of_defenders - 1
         self._remove_wall_defenders()
-        self.add_subbehavior(
-            skills.pivot_kick.
-            PivotKick(),  # TODO figure out what to do in scramble
-            name="robotCapture")
+        if self.contest_ball:
```

Comment:
I don't know if this is the place for this. I don't know if we should make the play implement the contesting ball portion. It's definitely a convenience but the init is a mile long.

Reply:
Removed it